static/resources/js/samaritan-common.js

```diff
@@ -128,13 +128,13 @@ function getURLParameter(sParam){
 // function to see if a date is inserted in a valid format
 
 function dateFormatValidator(date){
-    patt = /\d\d.\d\d.\d\d\d\d/i
+    patt = /\d\d.\d\d.\d\d\d\d/i;
     if (date.search(patt) == -1){
 
-        return "False"
+        return false
```

Comment:
don't forget `;`
end every statement in a semicolon. The reason for this is because not all browsers are equal and some browsers will treat that as a syntax error. Which is correct because javascript language definition requires all statements to end in a semicolon. The problem is that some browsers are more lenient and don't care about the semicolons as much as others do but we want to support as many browsers and be as consistent with our syntax as we can.